d3a_interface/schemas.py

```diff
@@ -121,6 +122,27 @@ class ScenarioSchemas:
                                                      {'type': 'string'}]}
                 }
             },
+            'home_meter': {
+                'type': 'object',
+                'properties': {
```

Comment:
I dont see `home_meter_profile` and `home_meter_profile_uuid` in here. Is this wanted?

Reply:
Good catch. I'm actually wondering though: the profile uuid is also not present in the other devices (with the exception of the Load). The validations don't fail because we don't declare any attribute as required (except `name`) but maybe this should be changed to make the validations stricter.

Reply:
Yeah, I think this was done on purpose (not expecting all values to be present) because the Class API on the d3a side is not expecting these values either. AFAIR, these schemas where introduced to make sure that the right parameter types are sent to the d3a. But yes, in the future we could be more strickt.